### PR TITLE
ETS-229 Initial API

### PR DESCRIPTION
**Pull request by simonbu11**
Functions search endpoint that will query learning providers in Azure Search. Includes tool to load public GIAS Establishment download into Azure Search

### CODE REVIEW

File: src/Dfe.Spi.Search.Functions.UnitTests/Dfe.Spi.Search.Functions.UnitTests.csproj [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <ItemGroup>
+      <ProjectReference Include="..\Dfe.Spi.Search.Application\Dfe.Spi.Search.Application.csproj" />
+      <ProjectReference Include="..\Dfe.Spi.Search.Functions\Dfe.Spi.Search.Functions.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <Reference Include="Microsoft.AspNetCore.Http.Abstractions, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60">
+        <HintPath>..\..\..\..\..\Library\Preferences\Rider2019.2\azure-functions-coretools\2.7.1846\Microsoft.AspNetCore.Http.Abstractions.dll</HintPath>
+      </Reference>
+    </ItemGroup>
```
Comment (mmiddleton3301) [inline src/Dfe.Spi.Search.Functions.UnitTests/Dfe.Spi.Search.Functions.UnitTests.csproj]: As mentioned on previous review, not sure this looks right? Could be missing some context around this...

File: src/GiasDataLoader/CommandLineOptions.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+using CommandLine;
```
Comment (mmiddleton3301) [inline src/GiasDataLoader/CommandLineOptions.cs:1]: A nice lib. 👍

**Review by mmiddleton3301 — CHANGES_REQUESTED** [review]
Looks mostly good, just one little bit to look into.

File: src/Dfe.Spi.Search.Functions.UnitTests/Dfe.Spi.Search.Functions.UnitTests.csproj (hunk 1 above)
Comment (simonbu11) [reply src/Dfe.Spi.Search.Functions.UnitTests/Dfe.Spi.Search.Functions.UnitTests.csproj]: Sorted. I am obviously selecting wrong import option in Rider sometimes

File: src/GiasDataLoader/CommandLineOptions.cs (hunk 2 above)
Comment (simonbu11) [reply src/GiasDataLoader/CommandLineOptions.cs:1]: Yeah, I like it when I need to do CLI things like this

**Review by mmiddleton3301 — APPROVED** [review]
👍

**Pull request merged**